web/style/nav.css

```diff
@@ -0,0 +1,17 @@
+nav.navbar {
+  position: fixed;
+  width: 100%;
+  background-color: #111;
+  padding: 6px 0;
+  color: white;
+}
+
+.nav-item {
+  padding: 6px 12px !important;
```

Comment:
In general I'd advise for careful use of `!important`, so these drew my attention. Is it because the `nav-item` is a class in bootstrap as well? Would defining our own class with a different name that applies this style, and using it alongside bootstrap's `nav-item` solve the need for `!important`?

Reply:
Ah sure - I will keep the use of `!important` in mind. 

In this case, I think if I'm more specific with the css selector (like `nav.navbar .nav-item` instead of just `.nav-item`) I wouldn't need to use `!imporant`

Fixed